cli/users.go

```diff
@@ -138,13 +140,15 @@ var cmdUsers = []cobra.Command{
 		},
 	},
 	{
-		Use:   "update [<user_id> <JSON_string> | tags <user_id> <tags> | identity <user_id> <identity> ] <user_auth_token>",
+		Use:   "update [<user_id> <JSON_string> | tags <user_id> <tags> | username <user_id> <username> | email <user_id> <email>] <user_auth_token>",
 		Short: "Update user",
-		Long: "Updates either user name and metadata or user tags or user identity\n" +
+		Long: "Updates either user name and metadata or user tags or user email\n" +
 			"Usage:\n" +
-			"\tmagistrala-cli users update <user_id> '{\"name\":\"new name\", \"metadata\":{\"key\": \"value\"}}' $USERTOKEN - updates user name and metadata\n" +
+			"\tmagistrala-cli users update <user_id> '{\"first_name\":\"new first_name\", \"metadata\":{\"key\": \"value\"}}' $USERTOKEN - updates user first and lastname and metadata\n" +
 			"\tmagistrala-cli users update tags <user_id> '[\"tag1\", \"tag2\"]' $USERTOKEN - updates user tags\n" +
-			"\tmagistrala-cli users update identity <user_id> newidentity@example.com $USERTOKEN - updates user identity\n",
+			"\tmagistrala-cli users update username <user_id> newusername $USERTOKEN - updates user name\n" +
+			"\tmagistrala-cli users update email <user_id> newemail@example.com $USERTOKEN - updates user email\n",
```

Comment:
Let's always use `magistrala-cli users <user_id> update <type_of_update>` so we don't confuse the end-users. I.e. `mg-cli <entity_type> <entity_id_where_we_need_id> <operation>`.

Reply:
Here is the issue related: https://github.com/absmach/magistrala/issues/2495